content/customize-dock.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Dock (Move or Customize)
+description: >
+  How to move the Dock to different areas of the screen and customize other settings.
+keywords:
+  - Launcher
+  - Bottom
+  - Left
+  - Moving
+  - Dock
+  - Dash to Dock
+  - Pop Dock
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+
+redirect_from:
+  - /movelauncher
+  - /articles/move-launcher
+---
+
+Pop!\_OS and Ubuntu both contain an application dock on the bottom of the screen. You can adjust this preference, move the dock to the bottom of the screen, and more in <u>Settings</u>.
```

Comment:
>Pop!\_OS and Ubuntu both contain an application dock on the bottom of the screen.

It's on the left side for Ubuntu. I think they should either have their locations mentioned separately so that they are both correct, or the locations should be omitted entirely.

>move the dock to the bottom of the screen,

If the previous sentence says it's already on the bottom edge of the screen, it doesn't make sense to say that it can be moved to the bottom of the screen (it's already there). Something like "change the location of the dock" seems like it would fit better.